tools/db-compiler-lite/src/main/java/com/grab/databinding/stub/main.kt

```diff
@@ -16,6 +16,16 @@
 
 package com.grab.databinding.stub
 
+import io.bazel.Status
+import io.bazel.Worker
+
 fun main(args: Array<String>) {
-    BindingStubCommand().main(args)
+    Worker.from(args = args.toList()).run { args ->
+        try {
+            BindingStubCommand().main(args)
+            Status.Success
+        } catch (e: Exception) {
```

Comment:
Should we print stacktrace for this exception, it might be useful for any issue investigation

Reply:
Yeah agreed